src/reference/90-Developers-Guide/06-Datatype/00.md

```diff
@@ -5,12 +5,12 @@ out: Datatype.html
 Datatype
 --------
 
-sbt-datatype is a code generation library and an sbt autoplugin that generates
+`sbt-datatype` is a code generation library and an sbt autoplugin that generates
```

Comment:
I feel the same way but my additional habit is monospacing when using something in the specific context of referring to its artifact, which I think is why it was instinctive here. Same with `launcher-interface` below.